.github/workflows/tests-run.yml

```diff
@@ -26,12 +26,12 @@ jobs:
       - name: Checkout ${{ github.ref }} branch in ${{ github.repository }} repository.
         uses: actions/checkout@v3
 
-      - name: Check Dependencies
+      - name: List dependency versions.
```

Comment:
By changing this I proved that it actually picks up the same commit. The logs show the new name.